zerver/lib/message.py

```diff
@@ -531,7 +531,7 @@ def build_dict_from_raw_db_row(row: Dict[str, Any]) -> Dict[str, Any]:
         return MessageDict.build_message_dict(
             message_id=row["id"],
             last_edit_time=row["last_edit_time"],
-            edit_history_json=row["edit_history"],
+            edit_history_entries=row["edit_history"],
```

Comment:
This seems unrelated to the commit message it's in, which is about the migration?